lib/uktt/parser.rb

```diff
@@ -0,0 +1,41 @@
+require 'json'
+require_relative 'parser/json_api'
+
+module Uktt
+  class Parser
+    def initialize(body, format)
+      @body = body
+      @format = format
+    end
+
+    def parse
+      return @body if json?
```

Comment:
How do you feel about declaring this as an

```ruby
attr_reader :body
```

Reply:
Yeah, I teeter between using and not using them. I should just pick a lane lol